spec/features/helpers.rb

```diff
@@ -0,0 +1,6 @@
+module FeatureHelpers
+  def create_list(name)
+    fill_in 'Name', with: name
+    click_button 'Create new list'
+  end
```

Comment:
I like what you've done here and it works. In some projects people add a 'support' folder in `/spec/`. However most projects also use the gem `factory_bot` which is all about generating models you need for your tests.

Even in integration tests you can still reach into your rails namespace to do things like create models `List.create` etc and also asset things in the database. While it's not quite as 'pure' in the sense of everything being done through the browser, it can make it much faster to do setup for your tests this way. It also allows you to set up models that might not be possible to create through the interface.

Reply:
Thanks Andy. Hopefully I'll have time to write some more tests (and test `Item`), in which case I'll probably set up a factory. Good point on being able to use the ActionRecord methods!